downloader/src/main/kotlin/VersionControlSystem.kt

```diff
@@ -261,10 +261,22 @@ abstract class VersionControlSystem {
                 false
             }
 
-        if (!addGuessedRevision(pkg.id.name, pkg.id.version) && pkg.id.type == "NPM" && pkg.id.namespace.isNotEmpty()) {
-            // Fallback for Lerna workspaces when scoped packages combined with independent versioning are used, e.g.
-            // support Git tag of the format "@organisation/my-component@x.x.x".
-            addGuessedRevision("${pkg.id.namespace}/${pkg.id.name}", pkg.id.version)
+        if (!addGuessedRevision(pkg.id.name, pkg.id.version)) {
```

Comment:
Commit message:

> differes a bit

"differs a bit"

Reply:
> Note: Mapping symbolic names to versions in GoMod is complex [1,2,3].
> So, it would be better to let the original code / tool perform that
> mapping rather than re-implenting in ORT.
> 
> [1] https://golang.org/ref/mod#vcs-version
> [2] https://golang.org/ref/mod#vcs-pseudo
> [3] https://golang.org/ref/mod#vcs-branch

Shout-out to @vbasem, would you know of a way to do that in GoMod itself? Like a hidden sub-command option or so? 😉